src/components/TextCardWithIcon/TextCardWithIcon.tsx

```diff
@@ -45,6 +45,12 @@ export const TextCardWithIcon: React.FC<TextCardWithIconProps> = ({
     </Box>
   );
 
+  const ButtonElement = button && (
+    <Button mt="auto" size="lg" onClick={button.onClick} maxWidth="24">
+      {button.text}
+    </Button>
+  );
```

Comment:
```suggestion
  const ButtonElement = (
    <Button mt="auto" size="lg" onClick={button.onClick} maxWidth="24">
      {button?.text}
    </Button>
  );
```

What do you think about this? It's more elegant IMO. But I don't mind too much.

Reply:
this doesn't work unfortunately, it displays a button without text even if the button is not defined in slice